NitroxServer/Program.cs

```diff
@@ -26,7 +27,7 @@ static void Main(string[] args)
 
             while (true)
```

Comment:
If you add a `Server.IsRunning` to the `Server` class you can use is here to quit reading commands and exit/close the console. Then you can write a close command and shutdown the server.

Reply:
Closing server from this while loop may cause more issues than closing from the UdpServer thread

Reply:
It should wait for UdpServer to exit.

1. Wait for UdpServer to exit.
2. Wait for Server to exit.
3. Exit while loops in Program.cs and let it exit itself.